bioimageio/core/prediction_pipeline/_model_adapters/_onnx_model_adapter.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, *, bioimageio_model: nodes.Model, devices=List[str]):
         onnx_inputs = self._session.get_inputs()
         assert len(onnx_inputs) == 1, f"expected onnx model to have one input got {len(onnx_inputs)}"
         self._input_name = onnx_inputs[0].name
+        # TODO onnx device management
```

Comment:
what happens with empty devices? Does it run on cpu? In any case maybe one should issue a warning here. (same goes for `TensorflowModelAdapterBase`).

Reply:
The empty device list is just a place holder until someone implements proper device management here... I think in both cases (tf and onnx) it runs on CPU. 
I agree, we should at least add a warning that arguments are ignored (if the devices are not None).

Reply:
I added warnings for this.